scrapy/commands/startproject.py

```diff
@@ -43,41 +44,68 @@ def _module_exists(module_name):
         if not re.search(r'^[_a-zA-Z]\w*$', project_name):
             print('Error: Project names must begin with a letter and contain'\
                     ' only\nletters, numbers and underscores')
-        elif exists(project_name):
-            print('Error: Directory %r already exists' % project_name)
         elif _module_exists(project_name):
             print('Error: Module %r already exists' % project_name)
         else:
             return True
         return False
 
+    def _copytree(self, src, dst):
```

Comment:
Ah yeah, you're right.
The problem is that `os.makedirs` call that is always called and requires the directory not to exist.
Can you perhaps add a docstring explaining this, then?